packages/babel-parser/src/plugins/flow.js

```diff
@@ -95,12 +95,26 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       return this.getPluginOption("flow", "all") || this.flowPragma === "flow";
     }
 
+    finishNodeAt<T: N.Node>(
```

Comment:
Would we be able to override `finishToken` to get the behavior you're looking for of bailing after a semi/string token?

Reply:
hmm.. interesting, let me try that. But which means it would have to handle whitespace, slash tokens for comments right?